lib/fly-ruby/configuration.rb

```diff
@@ -46,12 +46,12 @@ def initialize
 
     def database_uri
       @database_uri ||= URI.parse(database_url)
-      @database_uri
     end
 
     def regional_database_url
       uri = database_uri.dup
       uri.host = regional_database_host
+      uri.port = 5433 if in_secondary_region?
       uri.to_s
     end
```

Comment:
Is `if in_secondary_region?` both necessary and sufficient for port 5433?  Or should we specify via an explicit argument?  e.g.

```suggestion
    def regional_database_url(read_replica: false)
      uri = database_uri.dup
      uri.host = regional_database_host
      uri.port = 5433 if read_replica
      uri.to_s
    end
```

Reply:
That could work. This should probably be customised a bit more to not only work for PG. See #19 which adds support for LiteFS (sqlite), which will not require hijacking at all, and we may want different ports for different DBs like Mysql.